### DISCUSSION TITLE
自启动失效

### DISCUSSION THREAD

**Discussion by MKStoler2048** [ClassIsland/ClassIsland#319] [Q&A] [answered]
自启动失效，24H2，普通用户，禁用了UAC，还加载了uiaccess插件。

改成通过任务计划提升权限自启动？

**Comment by HelloWRC** [comment, marked as answer]
对，如果要在开机自启时以管理员身份运行 ClassIsland，需要使用计划任务，并以提升的权限运行。

**Reply by MKStoler4096** [reply]
好嘞（）
话说不能给设置里面加个“以管理员身份运行”，让ClassIsland自己创建任务计划，在启动时进行调用提升权限？

**Comment by unDefFtr** [comment]
> 对，如果要在开机自启时以管理员身份运行 ClassIsland，需要使用计划任务，并以提升的权限运行。

草 那只能重装系统了（fk学校啥都禁用

**Reply by Makitoid** [reply]
？你要在学校大屏重装系统？
> 草 那只能重装系统了（fk学校啥都禁用

**Reply by MKStoler4096** [reply]
其实也不是不行（）
